Src/BootCamp.Chapter/ToggleableGridJagged.cs

```diff
@@ -1,18 +1,52 @@
-﻿using System;
+﻿ using System;
+using System.Text; 
 
 namespace BootCamp.Chapter
 {
     public class ToggleableGridJagged : IToggleableGrid
     {
         private readonly IGridClearer _gridClearer;
+        private readonly bool[][] _toggle;
 
         public ToggleableGridJagged(bool[][] toggles, IGridClearer gridClearer)
         {
+            _toggle = toggles;
             _gridClearer = gridClearer;
         }
 
         public void Toggle(int x, int y)
         {
+            _toggle[x][y] = !_toggle[x][y];
+            StringBuilder sb = new StringBuilder();            
```

Comment:
Move this close to where it's used- next to loop

Reply:
I leave the line 19 at Toggle function and make Display function only judge element value( true/false) of array.
High/Low level function is a great strategy, I should focus on code reuse. 
Have a good day~